packages/app/src/components/ErrorPages/NotFoundErrorPage.tsx

```diff
@@ -0,0 +1,36 @@
+import type { AppComponents } from '@backstage/core-plugin-api';
+
+import Button from '@mui/material/Button';
+
+import { ErrorPage } from './ErrorPage';
+
+export const NotFoundErrorPage: AppComponents['NotFoundErrorPage'] = ({
+  children,
+}) => (
+  <ErrorPage
+    title={
+      <>
+        <strong>404</strong> We couldn't find that page
+      </>
+    }
+    message={
+      <>
+        Try adding an <strong>index.md</strong> file in the root of the docs
+        directory of this repository.
```

Comment:
Hmm, this is a generic error page but it talks about tech docs.

We should have a more generic message here.

If you want have a message for techdocs as well, then you can check the current pathname of course, but we should show this only when we're sure that it's a "good recommendation".

This messages could be used for example if the path matches one of these two:

* /docs/...
* /catalog/:namespace/:kind/:name/docs/...

cc @ShiranHi